java/code/src/com/suse/manager/webui/websocket/WebsocketSessionConfigurator.java

```diff
@@ -35,12 +34,12 @@ public void modifyHandshake(ServerEndpointConfig config,
             HandshakeRequest request,
             HandshakeResponse response) {
         HttpSession httpSession = (HttpSession)request.getHttpSession();
-        HttpServletRequest sr = (HttpServletRequest) httpSession.getAttribute("__original_request__");
-        User user = new RequestContext(sr).getCurrentUser();
-        // force roles to be loaded so now since it will fail from the websocket thread
-        if (user != null) {
-            user.getRoles();
-            config.getUserProperties().put("currentUser", user);
-        }
+        Optional.ofNullable(httpSession.getAttribute("webUserID"))
```

Comment:
Since we are not using the httpSession.getAttribute("__original_request__") anymore, we can also remove the attribute setting from the filter, right ?

Reply:
Hey Luis, I think I had remove it: https://github.com/uyuni-project/uyuni/pull/1235/files#diff-deb3f46949f5384c42d5f5224a1583a0L81

Reply:
oh you are right :D It seems I overlooked that eheh